lms/djangoapps/courseware/module_render.py

```diff
@@ -191,12 +191,14 @@ def toc_for_course(user, request, course, active_chapter, active_section, field_
                     }
 
                     #
-                    # Add in rendering context for proctored exams
+                    # Add in rendering context for proctored exams and timed exams
                     # if applicable
                     #
                     is_proctored_enabled = (
-                        getattr(section, 'is_proctored_enabled', False) and
-                        settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False)
+                        (
+                            getattr(section, 'is_proctored_enabled', False) or
+                            getattr(section, 'is_time_limited', False)
+                        ) and settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False)
```

Comment:
Just check for 'is_time_limited' as all proctored exams must be time limited.

Also, we should write some unit tests (there should be some for proctored exams) to confirm that timed only exams also get the navigation bar information set.